react-frontend/src/core-components/label/LabelComponent.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+const LabelComponent = ({ text, className }) => (
```

Comment:
as discussed we can use the same approach here as well just like we did for the grid, create a form directory and export form because we aren't adding any custom feature to label

also remove the test file

cc: @sahilbhatia